src/events.rs

```diff
@@ -248,6 +248,13 @@ pub enum EventType {
         timer: EphemeralTimer,
     },
 
+    // Chatlist changed and ui should reload it (reordering, entry added or removed)
+    //
+    // This event is fired when the ordering or contents of the chatlist change,
+    // for changes in individual chat list items listen to `IncomingMsg`, `MsgsNoticed`, `MsgDelivered`, `MsgFailed`, `MsgRead` and `ChatModified`
```

Comment:
Does "or contents" mean that the event is also fired if something within a chatlist item changed (like, a read receipt was received for the last message you sent and now two green checkmarks are shown)

Reply:
no, I'm open for suggestion on how to make this more clear